backend/tracim_backend/config.py

```diff
@@ -167,7 +169,8 @@ def _load_global_config(self) -> None:
             "contents/file,"
             "contents/html-document,"
             "contents/folder,"
-            "agenda"
+            "agenda,"
+            "office_document"
```

Comment:
I'l like to rename this to `online-document-edition` ou `collaborative-document-edition`

Reply:
not a problem, but this need frontend to be aware of this @basilelegal .